spec/spec_helper.rb

```diff
@@ -133,6 +133,20 @@ def client_assertion_with_nonexistent_client_id_path
   end
 
   config.include FactoryGirl::Syntax::Methods
+
+  config.include JSON::SchemaMatchers
+  config.json_schemas[:archive_schema] = "lib/schemas/archive-format.json"
+
+  # rubocop:disable Rails/DynamicFindBy
+  gemdir = Gem::Specification.find_by_name("diaspora_federation-json_schema").gem_dir
```

Comment:
I just created diaspora/diaspora_federation#70 and I think that's a cleaner solution for that, because you can just do this now:
```ruby
  JSON::Validator.add_schema(
    JSON::Schema.new(
      DiasporaFederation::Schemas.federation_entities,
      Addressable::URI.parse(DiasporaFederation::Schemas::FEDERATION_ENTITIES_URI)
    )
  )
```

What do you think about it?